integrations/optimizely/lib/index.js

```diff
@@ -51,38 +47,29 @@ Optimizely.prototype.initialize = function() {
     type: 'integration',
     OAuthClientId: '5360906403'
   });
-  // Initialize listeners for both Classic and New Optimizely
```

Comment:
nit: `var self = this;` not used anymore. can delete

Reply:
I think I just brought this back 🙈